src/features/presence/PresenceCollaborators.scss

```diff
@@ -0,0 +1,45 @@
+@import './styles';
+
+.bdl-PresenceCollaborators-item {
+    display: flex;
+    align-items: center;
```

Comment:
We usually want to allow the parent component to lay out its children. That way, this component can be reused in a variety of contexts.

Reply:
Updated!